parent/pom.xml

```diff
@@ -89,8 +89,8 @@
         <cxf.atmosphere.version>2.5.2</cxf.atmosphere.version>
         <cxf.bcprov.version>1.64</cxf.bcprov.version>
         <cxf.brave.reporter.version>2.12.1</cxf.brave.reporter.version>
-        <cxf.brave.version>5.9.4</cxf.brave.version>
-        <cxf.brave.zipkin.version>2.19.3</cxf.brave.zipkin.version>
+        <cxf.brave.version>5.10.1</cxf.brave.version>
+        <cxf.brave.zipkin.version>2.20.1</cxf.brave.zipkin.version>
```

Comment:
According to the parent pom for 5.10.1, the Zipkin version is still 2.19.3, so I guess we should stay aligned with that: https://repo.maven.apache.org/maven2/io/zipkin/brave/brave-parent/5.10.1/brave-parent-5.10.1.pom

Reply:
it doesn't matter, but no problem reverting to that.